go.mod

```diff
@@ -5,16 +5,19 @@ go 1.12
 // Che Plugin Broker branch devworkspace-controller
 require github.com/eclipse/che-plugin-broker v3.1.1-0.20200207223144-b20597f15e4c+incompatible
 
+// use-devfile-2.0-in-workspace-controller
+require github.com/devfile/kubernetes-api v0.0.0-20200603111415-8e1485401159
+
+// replace github.com/devfile/kubernetes-api => /home/dfestal/go/src/github.com/devfile/kubernetes-api
```

Comment:
Accidental inclusion? 

It's possible to point at a branch pushed remotely, which we do e.g. for the plugin broker above.